README.md

```diff
@@ -1,52 +1,67 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# First Capstone Project: Remote Experts Website
 
-# Portfolio Application
-
-> My first portfolio project which is mobile first responsve website
+Description of the project:
+This project represents the mobile version and the Desktop version of the first capstone project from Microverse, Where I used all the skills and knowledge that I acquired form the first module.
```

Comment:
- [ ] [REQUIRED] You have done a great job implementing a well descriptive README, however, one of the requirements states that you should record a 5 minutes video presenting the features of the project. Kindly ensure that the video contains a clear description of the project in English. Also, it should contain a demo of the project, showing the pages in different screen sizes and a highlight about a piece of the code that you are proud of making. Kindly attach the link to the loom video to the README file.
- [ ] [REQUIRED] Kindly use Github Pages to deploy the webpages and attach a link to the deployment page to the README file.

Reply:
eslint, I already included it in my project

Reply:
![Screenshot from 2022-03-18 19-24-09](https://user-images.githubusercontent.com/58692590/159062049-d87ca7b6-11e1-431b-9e2f-861e26c8c56e.png)
Dear @Melaku05  You currently do not have `eslint` checks. Please ensure that `eslint` is correctly installed in your `main` branch so it can be compared with the capstone branch. This is why you can see the `eslint` locally but it doesn't show on Github.